extraction/text_extractor.py

```diff
@@ -0,0 +1,41 @@
+"""This script checks whether DOM has text tag or not and
```

Comment:
You can create IExtractor which is interface for any extractor.
```
class IExtractor:
    def extract(self, ....) -> :
        """Extract content from BeautifulSoup tags."""
        pass
```

You can have TextExtractor implement IExtractor

Reply:
Implemented in next iteration. Thanks for review.